actors/builtin/miner/miner_test.go

```diff
@@ -1362,73 +1362,66 @@ func TestExtendSectorExpiration(t *testing.T) {
-	//	var initialLockedFunds abi.TokenAmount
-	//
-	//	// A miner will pay the minimum of termination fee and locked funds. Add some locked funds to ensure
-	//	// correct fee calculation is used.
-	//	actor.addLockedFund(rt, big.NewInt(1<<61))
-	//
-	//	{
-	//		// Verify that a sector expiration was registered.
-	//		st := getState(rt)
-	//		expiration, err := st.GetSectorExpirations(rt.AdtStore(), sector.Expiration)
-	//		require.NoError(t, err)
-	//		expiringSectorNos, err := expiration.All(1)
-	//		require.NoError(t, err)
-	//		assert.Len(t, expiringSectorNos, 1)
-	//		assert.Equal(t, sector.SectorNumber, abi.SectorNumber(expiringSectorNos[0]))
-	//		initialLockedFunds = st.LockedFunds
-	//	}
-	//
-	//	sectorSize, err := sector.SealProof.SectorSize()
-	//	require.NoError(t, err)
-	//	sectorPower := miner.QAPowerForSector(sectorSize, sector)
-	//	sectorAge := rt.Epoch() - sector.Activation
-	//	expectedFee := miner.PledgePenaltyForTermination(sector.InitialPledge, sectorAge, actor.epochReward, actor.networkQAPower, sectorPower)
-	//
-	//	sectors := bitfield.New()
-	//	sectors.Set(uint64(sector.SectorNumber))
-	//	actor.terminateSectors(rt, &sectors, expectedFee)
-	//
-	//	{
-	//		st := getState(rt)
-	//
-	//		// expect sector expiration to have been removed
-	//		err = st.ForEachSectorExpiration(rt.AdtStore(), func(expiry abi.ChainEpoch, sectors *abi.BitField) error {
-	//			assert.Fail(t, "did not expect to find a sector expiration, found expiration at %s", expiry)
-	//			return nil
-	//		})
-	//		assert.NoError(t, err)
-	//
-	//		// expect sector to have been removed
-	//		_, found, err := st.GetSector(rt.AdtStore(), sector.SectorNumber)
-	//		require.NoError(t, err)
-	//		assert.False(t, found)
-	//
-	//		// expect fee to have been unlocked and burnt
-	//		assert.Equal(t, big.Sub(initialLockedFunds, expectedFee), st.LockedFunds)
-	//
-	//		// expect pledge requirement to have been decremented
-	//		assert.Equal(t, big.Zero(), st.InitialPledgeRequirement)
-	//	}
-	//})
+	commitSector := func(t *testing.T, rt *mock.Runtime) *miner.SectorOnChainInfo {
+		actor.constructAndVerify(rt)
+		precommitEpoch := abi.ChainEpoch(1)
+		rt.SetEpoch(precommitEpoch)
+		sectorInfo := actor.commitAndProveSectors(rt, 1, 100, nil)
+		return sectorInfo[0]
+	}
+
+	t.Run("removes sector with correct accounting", func(t *testing.T) {
+		rt := builder.Build(t)
+		sector := commitSector(t, rt)
+
+		// A miner will pay the minimum of termination fee and locked funds. Add some locked funds to ensure
+		// correct fee calculation is used.
+		actor.addLockedFund(rt, big.Mul(big.NewInt(1e18), big.NewInt(20000)))
+		st := getState(rt)
+		initialLockedFunds := st.LockedFunds
+
+		sectorSize, err := sector.SealProof.SectorSize()
+		require.NoError(t, err)
+		sectorPower := miner.QAPowerForSector(sectorSize, sector)
+		dayReward := miner.ExpectedDayRewardForPower(actor.epochReward, actor.networkQAPower, sectorPower)
+		sectorAge := rt.Epoch() - sector.Activation
```

Comment:
It's not clear that this is definitely >0, so roll the epoch forward a bit after the `commitSector`.

Reply:
Moved forward 100 after commitSector